content/javascript/concepts/promise/terms/all/all.md

```diff
@@ -11,7 +15,11 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-The `.all()` method returns a new Promise that can be accessed as an array of resolved values of fulfulled Promises. It takes an iterable object, such as an `Array`, that contains one or more Promise objects. This is ideal when working with Promises that depend on one another's completion.
+The .all() method in JavaScript Promises helps us work with multiple tasks at once. It takes a group of tasks (Promises) and returns a new Promise.
+
+When we use .all(), all the tasks start together, and .all() waits for all of them to finish. It gives us a Promise that contains the results of all the tasks.
```

Comment:
[Proofreading/Nitpick] These two lines share the same issue as line 5. Refer to [my feedback from there](https://github.com/Codecademy/docs/pull/2459/files#r1225974895) to help you.
```suggestion
The .all() method in JavaScript Promises helps us work with multiple tasks at once. It takes a group of tasks (Promises) and returns a new Promise.


When we use .all(), all the tasks start together, and .all() waits for all of them to finish. It gives us a Promise that contains the results of all the tasks.
```